packages/dashboard/README.md

```diff
@@ -1,3 +1,5 @@
+# Dashboard
+
```

Comment:
Let's remove this from the PR once this is ready to be merged, we can put it back later if we need to update the md file but I'd like to keep the changes limited to only required files.

```suggestion
```

Reply:
These were added to prove that the linters are properly triggered.